nautobot/dcim/filters.py

```diff
@@ -1055,6 +1077,10 @@ class DeviceComponentFilterSet(CustomFieldModelFilterSet):
 # TODO: should be CableTerminationFilterSetMixin
 class CableTerminationFilterSet(django_filters.FilterSet):
     cabled = django_filters.BooleanFilter(field_name="cable", lookup_expr="isnull", exclude=True)
+    cable = django_filters.ModelMultipleChoiceFilter(
```

Comment:
I've been trying to stick with `slug` when available and falling back to `name`. Name is a mix of unique/indexed depending on the model. Could filtering on non-indexed fields impact performance for dynamic form generation? 

Are we planning to add slugs to all models in 2.0 so we can standardize on that as our natural key?

Reply:
Adding true natural keys for all models is highly desirable, but they may not always be slugs (e.g. what would the slug of an IPAddress be? Would make more sense for its natural key to be something like (address, VRF) rather than adding a separate slug field). I think probably leaving this specific instance alone for now is fine.